packages/video_player/video_player_web/pubspec.yaml

```diff
@@ -21,7 +22,8 @@ dependencies:
     sdk: flutter
   flutter_web_plugins:
     sdk: flutter
-  video_player_platform_interface: ">=4.2.0 <6.0.0"
+  video_player_platform_interface:
+    path: ../video_player_platform_interface
```

Comment:
Why are you changing the web package at all?

Reply:
I was expecting to also change the web video player, of course I can definitely roll back this change